src/pages/index.tsx

```diff
@@ -1,7 +1,30 @@
+import IndexContents from 'components/index';
 import React from 'react';
+import styled from 'styled-components';
+import OceanBackground from 'components/index/OceanBackground';
+import qs from 'qs';
+const IndexPage = ({location}) => {
+  const query = qs.parse(location.search, {
+    ignoreQueryPrefix: true
+  });
+  
+  const data = query.q
+  console.log('name: '+ data);
 
-const IndexPage = () => {
-  return <>Index</>;
+  const isValid = (data === undefined) ? false : true
```

Comment:
```suggestion
  const isValid = data !== undefined
```

이렇게 쓰면 더 깔끔할거 같은데 어떠세용?